marshmallow/fields.py

```diff
@@ -1344,6 +1339,45 @@ def _deserialize(self, value, *args, **kwargs):
         return self.constant
 
 
+class Default(Field):
```

Comment:
`fields.Inferred` might be a better name

we could also define this in `schema.py`; this is really more of a private utility and should not be directly used

Reply:
+1 for renaming to `fields.Inferred`. I don't feel strongly about where this goes, but it should be documented that this should be considered private. Perhaps add this to the docstring:

```
.. warning::

    This class is treated as private API.
    Users should not need to use this class directly.
```